upgrade-cluster.yml

```diff
@@ -0,0 +1,33 @@
+---
+- hosts: all
+  any_errors_fatal: true
+  gather_facts: true
+
+- hosts: all:!network-storage
+  any_errors_fatal: true
+  roles:
+    - { role: kubernetes/preinstall, tags: preinstall }
+
+- hosts: etcd:!k8s-cluster
+  any_errors_fatal: true
+  serial: 1
+  roles:
+    - { role: etcd, tags: etcd }
+
+- hosts: kube-node
+  any_errors_fatal: true
+  serial: 1
+  roles:
+    - { role: etcd, tags: etcd }
+    - { role: upgrade/pre-upgrade, tags: upgrade/pre-upgrade }
+    - { role: kubernetes/node, tags: node }
+    - { role: network_plugin, tags: network }
+    - { role: upgrade/post-upgrade, tags: upgrade/post-upgrade }
+
+- hosts: kube-master
+  any_errors_fatal: true
+  serial: 1
+  roles:
+    - { role: etcd, tags: etcd }
+    - { role: kubernetes/node, tags: node }
+    - { role: kubernetes/master, tags: master }
```

Comment:
The apps should already be deployed, no? As part of the initial install? We're just trying to bump the etcd, network_plugin, or k8s container versions I thought.